.travis.yml

```diff
@@ -14,8 +14,14 @@ addons:
       - debian-sid
     packages:
       - shellcheck
-install: tests/install.sh
-script: tests/${TEST}
+install:
+  - tests/install.sh
+  - pip install pytest-cov
```

Comment:
Why not add this to tests/install.sh instead of adding it here? Then we get the benefit of shell linting from shell check, and it can be partnered with the existing pip commands.